Helmsman/hooks/cleanup-partner-data.sh

```
@@ -0,0 +1,166 @@
+# Function to execute SQL
+execute_sql() {
+  local database=$1
+  local sql=$2
+  local description=$3
+  
+  if [ "$DRY_RUN" == "true" ]; then
+    echo "[DRY-RUN] $description"
+    echo "  SQL: $sql"
+    # Show count that would be deleted - convert DELETE to SELECT COUNT(*)
+    local count_sql="${sql/DELETE/SELECT COUNT(*)}"
+    local count=$(PGPASSWORD="$DB_PASSWORD" psql -h "$DB_HOST" -p "$DB_PORT" -U "$DB_USER" -d "$database" -t -c "$count_sql" 2>/dev/null | xargs || echo "0")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)